app/boards/shields/splitreus62/splitreus62.conf

```diff
@@ -0,0 +1,2 @@
+# Uncomment the following line to enable the Lily58 OLED Display
+# CONFIG_ZMK_DISPLAY=y
```

Comment:
Same thing here. Does it support a display? (also this isn't the Lily58)

Reply:
It does not. And it is not.

Reply:
Removed